packages/angular/src/tracing.ts

```diff
@@ -101,6 +104,40 @@ export class TraceService implements OnDestroy {
     }),
   );
 
+  // The ResolveEnd event is fired when the Angular router has resolved the URL
+  // and activated the route. It holds the new resolved router state and the
+  // new URL.
+  // Only After this event, the route is activated, meaning that the transaction
+  // can be updated with the parameterized route name before e.g. the route's root
+  // component is initialized. This should be early enough before outgoing requests
+  // are made from the new route.
+  // In any case, this is the earliest stage where we can reliably get a resolved
+  //
```

Comment:
I think this comment was truncated by accident

Reply:
Thanks; adjusted the comment a little because it generally wasn't up to date with the latest changes